.circleci/config.yml

```diff
@@ -513,6 +525,8 @@ jobs:
       - MAGMA_ROOT=/home/circleci/project
     steps:
       - checkout
+      - build/determinator:
+          paths: "feg lte cwf"
```

Comment:
same here?

Reply:
ya +orc8r, we probably do not need cwf ... i think